packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx

```diff
@@ -416,18 +444,31 @@ export const VideoForm: React.FC<VideoFormProps> = React.memo(({ onSubmit, setFo
         <Controller
           name="title"
           control={control}
-          rules={textFieldValidation({ name: 'Video Title', minLength: 3, maxLength: 60, required: true })}
+          rules={textFieldValidation({
+            name: 'Video title',
+            minLength: MIN_TITLE_LENGTH,
+            maxLength: MAX_TITLE_LENGTH,
+            required: true,
+          })}
           render={({ field: { value, onChange } }) => (
             <StyledTitleArea
+              ref={titleInputRef}
               onChange={onChange}
               value={value}
-              min={3}
-              max={60}
-              placeholder="Video title"
+              min={MIN_TITLE_LENGTH}
+              max={MAX_TITLE_LENGTH}
+              placeholder="Enter video title"
               disabled={videoFieldsLocked}
+              error={!!errors.title}
+              onFocus={() => setTitleTooltipVisible(false)}
+              onBlur={() => setTitleTooltipVisible(true)}
+              helperText={errors.title && errors.title.message}
             />
           )}
         />
+        {titleTooltipVisible && (
+          <Tooltip text="Click to edit" arrowDisabled placement="top-start" reference={titleInputRef} />
+        )}
```

Comment:
Why not just wrap `StyledTitleArea` with the tooltip component?

Reply:
Because of the flag `titleTooltipVisible`. When input is active, tooltip shouldn't be visible. Do you know easier way to achieve this?

Reply:
nop